src/rules/no-test-prefixes.ts

```diff
@@ -43,12 +49,14 @@ export default createRule({
 function getPreferredNodeName(nodeName: string) {
```

Comment:
nit: a good refactor for this could be to do `split('.')` - then you could just splice the `only`/`skip` in as element 1, and return `.join('.')`.

I'm fine with this how it is, so that doesn't have to be done - just thinking of the future :)

Reply:
I think I'll leave this one as-is